README.md

```diff
@@ -1,6 +1,6 @@
 # My-Portfolio
 
-> This is an HTML project about my portfolio, here comes the first 2 sections, then the projects section where you can see a few projects,about me section and the form section.
+> This is an HTML project about my portfolio, here comes the first 2 sections, then the projects section where you can see a few projects,about me section and the form section, this porject is responsive for a 375px screen(mobile screen) and a 1021px screen(desktop screen).
```

Comment:
- [ ] [OPTIONAL] MIT.md license file is missing so please add it 📃
- [ ] [OPTIONAL] I also recommend adding a screenshot to the readme file 📷
- [ ] [OPTIONAL] Let's also remove the unused parts from the readme like _Prerequisites, Setup, Install, Usage, Run tests, Deployment_